front-end/src/components/PostDetailModal.tsx

```diff
@@ -10,13 +11,23 @@ interface Props {
 
 export default function PostListItem(props:Props) {
   const post: Post = props.post;
+
+  function Content() {
```

Comment:
I think it's pretty clever to use functions like this over what I'm used to, which is just changing around scoped variables 👍

Reply:
Good to know! I feel like I have no idea what I'm doing when it comes to React haha